Updates from Kristine's science writer for simpler language (#525)
Sage-Bionetworks/synapseDocs: articles/making_a_project.md

---

# Projects in Synapse 

> `Projects` in Synapse serve as workspaces where researchers collaborate.  Researchers using Synapse can work in private or collaborate in groups. Some users share their research across large consortia. Projects support the following:

Reviewer: ```suggestion
`Projects` in Synapse serve as workspaces where researchers collaborate. Researchers using Synapse can work in private or collaborate in groups. Some users share their research across large consortia. Projects support the following:
```

* **Organize Your Work**: With Synapse you can organize the parts in your workflow (data, code, etc) into a hierarchy like a file system. With the tabs across the top of each project, you can easily navigate to your [Wikis](http://docs.synapse.org/articles/wikis.html), [Files and Folders](http://docs.synapse.org/articles/files_and_versioning.html), SQL-based [Tables](http://docs.synapse.org/articles/tables.html), and even conduct conversations using [Discussion Forums](http://docs.synapse.org/articles/discussion.html). 

> * **Store Data, Code, and Results**: You can upload your data, code, and results to Synapse, or store a reference to their location in your local computer system. Everything can be stored as Files, hosted by Synapse or in your own external cloud storage, or using [Docker Containers](http://docs.synapse.org/articles/docker.html). 

Reviewer: ```suggestion
* **Store Data, Code, and Results**: You can upload your data, code, and results to Synapse, or store a reference to their location in your local computer system or on the web. Everything can be stored as Files, hosted by Synapse, in your own external cloud storage, or using [Docker Containers](http://docs.synapse.org/articles/docker.html). 
```

> * **Control Access or Release Your Work Publicly**: You have complete control over how users and groups can interact with your work. Work privately or collaborate openly, and everything in between.

Reviewer: ```suggestion
* **Control Access or Release Your Work Publicly**: You have complete control over how users and groups can interact with your work. Work privately, openly, or somewhere in between.
```

* **Custom, Searchable Annotations**: Assign any key/value pair you want. Those values become searchable and available to those granted access.

* **Full Programmatic API and Python/R Integration**: Synapse is built on a RESTful API, so you can interact with Synapse using any programming language. We have built out full-featured clients for Python, R, and Java. 

* **Attach Figures and Documents**: Upload documents and images via the website or programmatically.

> * **Describe and Version**: Describe each of the elements in your workflow, and then practice version control when you see fit. This allows you to explicitly record which version of the code you ran on which version of the data.

Reviewer: ```suggestion
* **Describe and Version**: Describe each element in your workflow and practice version control when you see fit. This allows you to explicitly record which version of the code you ran on which version of the data.
```

> * **Create Links and See Usage**: You can make a link to store data from elsewhere on Synapse

Reviewer: ```suggestion
```

Reviewer: added above in the Files that you can link

> and use it in your project. Other projects using your data are displayed automatically.

Reviewer: ```suggestion
```

we don't do this?
